docs/WIP/Tutorial_Motif_only_Matrix_Profile.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        num_segments = n // dsr  # Number of segments after downsampling
</pre><p>Does this imply that we will always truncate any parts of the time series that don't fit within some multiple of <code>dsr</code>? Is that the what the paper is doing?</p><p> </p><p>Also, to be consistent with other parts of our code, we should rename <code>num_segments</code> to <code>n_chunks</code>     </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/1046/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p><strong>Regarding PAA</strong></p><p>In most of the implementations, PAA seems to truncate the time series if the length cannot be exactly divided by the sampling rate. For instance, see tslearn implementation here: </p><p> </p><p><a href="https://github.com/tslearn-team/tslearn/blob/99379466ec934fcf39dd4f30ec13c033f33f4a34/tslearn/piecewise/piecewise.py#L143-L153" target="_blank">https://github.com/tslearn-team/tslearn/blob/99379466ec934fcf39dd4f30ec13c033f33f4a34/tslearn/piecewise/piecewise.py#L143-L153</a></p><p> </p><p>However, I know that there is a paper[ref 1] in which the authors mentioned that they used a modified version of PAA to not truncate data based on another study [ref 2]</p><p> </p><p>[ref 1] Matrix Profile XXII: Exact Discovery of Time Series Motifs under DTW. <a href="https://arxiv.org/pdf/2009.07907" target="_blank">https://arxiv.org/pdf/2009.07907</a></p><p> </p><p>[ref 2]  Generalized SAX code&nbsp;to handle the&nbsp;<em>N/n not equal an integer</em>&nbsp;case, by Li Wei. Look it up here:</p><p><a href="https://www.cs.ucr.edu/~eamonn/SAX.htm" target="_blank">https://www.cs.ucr.edu/~eamonn/SAX.htm</a></p><p> </p><p> </p><p>The materials above can be visited later if needed.</p><p> </p><p> </p>

Reply:
<p>Alternatively, we can/should reach out to the original authors to see guidance. At the end of the day, what would impact does each choice have and what would a user expect?</p>